app/src/components/TextEditor.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="contact_text-editor">
+    <div class="md-layout md-alignment-center-right">
+        <button @click="applyHeading" class="md-button md-dense md-icon-button"><b style="font-size:15px">H</b></button>
```

Comment:
For mine locally, `applyHeading` allows me to apply H but not un-apply it

Reply:
I see, I'll take a look